stroeer/core/v1/shared.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package stroeer.core.v1;
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/core/v1;core";
+option java_multiple_files = true;
+option java_package = "de.stroeer.core.v1";
+
+
+// References to get access to the article on different technical platforms like
+// canonical web or AMP.
+message References {
+  // URL path for this article e.g. /section/id_$ID/title.html (required).
+  string path = 1;
+
+  // Canonical URL of the article, may differ if external, e.g. https://www.giga.de/external.html
+  // (required).
+  string canonical = 2;
+
+  // AMP URL of the article.
+  string amp = 3;
+}
+
+// Hierarchical section information of the article till root.
+//
+// For example:  /politik/deutschland/ -> /politik/ -> /
+message SectionTree {
```

Comment:
Some thoughts:
While I think about the internal link structure I thought about the main navigation and how it could be handled by our models. The `SectionTree` would be perfect for this when the name would be more generic.

Reply:
we won't need the hierarchy in the navigation though, do we? Still we could reuse this message - any naming ideas in mind @bobaaaaa ?

Reply:
The most obvious name would be:

```proto
message ReferenceTree {
  // Section path of the article, e.g. /politik/deutschland/ (required).
  Reference reference = 1;

  // Parent section. Usually points to the parent section, if there is none, e.g. when in `root` this value is `null`
  ReferenceTree parent = 2;
}
```

If we take it 100% correctly even `Tree` is the wrong postfix -- it's an `TreeItem` or `Leaf`. /cc @thisismana

Reply:
`LinkedReference` 🎉